From d81e1270694d5764e017580dc2c6bb2f84bae193 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Mon, 16 Mar 2020 09:53:59 +0800
Subject: [PATCH 1/2] configs/rockchip: rk3399pro: enable chromium needed
 configs for tests

Due to the ffmpeg's copyright, the sdk had to disable some configs.

They have the below content when enable them
1) BR2_PACKAGE_GST1_LIBAV=y
    The codecs, muxers, bitstreams and filters to build can be selected in the
    ffmpeg package.It depended on ffmpeg gstreamer1 gst1-plugins-base.

2)#include "demo.config"
    It's the multimedia video demo for tests.

3)#include "video_ffmpeg.config"
    Enable the ffmpeg with video

4)#include "chromium.config"
    Support the chromium browser

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 configs/rockchip/video_gst.config              | 1 +
 configs/rockchip_rk3399pro-multi-cam_defconfig | 3 +++
 configs/rockchip_rk3399pro_combine_defconfig   | 1 +
 configs/rockchip_rk3399pro_defconfig           | 3 +++
 4 files changed, 8 insertions(+)

diff --git a/configs/rockchip/video_gst.config b/configs/rockchip/video_gst.config
index 9e273a88bf..813fff8f16 100644
--- a/configs/rockchip/video_gst.config
+++ b/configs/rockchip/video_gst.config
@@ -24,3 +24,4 @@ BR2_PACKAGE_GST1_PLUGINS_UGLY_PLUGIN_ASFDEMUX=y
 BR2_PACKAGE_GST1_PLUGINS_UGLY_PLUGIN_DVDLPCMDEC=y
 BR2_PACKAGE_GST1_PLUGINS_UGLY_PLUGIN_DVDSUB=y
 BR2_PACKAGE_GST1_PLUGINS_UGLY_PLUGIN_MPEG2DEC=y
+BR2_PACKAGE_GST1_LIBAV=y
diff --git a/configs/rockchip_rk3399pro-multi-cam_defconfig b/configs/rockchip_rk3399pro-multi-cam_defconfig
index 9f48a4a60f..5fb3dbc6d4 100644
--- a/configs/rockchip_rk3399pro-multi-cam_defconfig
+++ b/configs/rockchip_rk3399pro-multi-cam_defconfig
@@ -1,8 +1,11 @@
 #include "rk3399pro_arm64.config"
 #include "base.config"
 #include "base_extra.config"
+#include "chromium.config"
+#include "demo.config"
 #include "gpu.config"
 #include "display.config"
+#include "video_ffmpeg.config"
 #include "video_mpp.config"
 #include "video_gst.config"
 #include "video_gst_rtsp.config"
diff --git a/configs/rockchip_rk3399pro_combine_defconfig b/configs/rockchip_rk3399pro_combine_defconfig
index 3eec7d086a..eee3716205 100644
--- a/configs/rockchip_rk3399pro_combine_defconfig
+++ b/configs/rockchip_rk3399pro_combine_defconfig
@@ -2,6 +2,7 @@
 #include "base.config"
 #include "base_extra.config"
 #include "chromium.config"
+#include "demo.config"
 #include "gpu.config"
 #include "display.config"
 #include "video_ffmpeg.config"
diff --git a/configs/rockchip_rk3399pro_defconfig b/configs/rockchip_rk3399pro_defconfig
index cc66a53c82..e9c91cc2ad 100644
--- a/configs/rockchip_rk3399pro_defconfig
+++ b/configs/rockchip_rk3399pro_defconfig
@@ -1,8 +1,11 @@
 #include "rk3399pro_arm64.config"
 #include "base.config"
 #include "base_extra.config"
+#include "chromium.config"
+#include "demo.config"
 #include "gpu.config"
 #include "display.config"
+#include "video_ffmpeg.config"
 #include "video_mpp.config"
 #include "video_gst.config"
 #include "video_gst_rtsp.config"
-- 
2.25.0

